config/environments/appliance.rb

```diff
@@ -9,5 +9,5 @@
   config.log_level = ENV['CONJUR_LOG_LEVEL'] || :info
   config.middleware.use Rack::RememberUuid
   config.audit_socket = '/run/conjur/audit.socket'
-  config.audit_database ||= 'postgres://:5433/audit'
```

Comment:
So we are overriding any value that config.audit_database has? what with application.rb change you made

Reply:
This setting take precedence, we are overriding the value of config.audit_database